components/server/src/installation-admin/installation-admin-controller.ts

```diff
@@ -43,46 +28,6 @@ export class InstallationAdminController {
             }
         });
 
-        const adminUserCreateLoginTokenRoute = "/admin-user/login-token/create";
-        app.post(adminUserCreateLoginTokenRoute, async (req: express.Request, res: express.Response) => {
-            const span = TraceContext.startSpan(adminUserCreateLoginTokenRoute);
-            const ctx = { span };
-
-            log.info(`${adminUserCreateLoginTokenRoute} received.`);
-            try {
-                // In case there is no/an empty key specified: Nobody should be able to call this so they are not able to guess values here
-                if (!this.config.admin.loginKey) {
-                    throw new Error("Cannot handle request");
-                }
-
-                // Unblock the admin-user: it's blocked initially!
-                const user = await this.userDb.findUserById(BUILTIN_INSTLLATION_ADMIN_USER_ID);
-                if (!user) {
-                    throw new Error("Cannot find builtin admin-user");
-                }
-                user.blocked = false;
```

Comment:
@easyCZ How is the admin-user unblocked in the new flow?

Just want to make sure this is not overlooked.